packages/devtools_app/lib/src/provider/provider_screen.dart

```diff
@@ -122,3 +137,42 @@ void showProviderErrorBanner(BuildContext context) {
         .build(context),
   );
 }
+
+class _StateInspectorSettingsDialog extends ConsumerWidget {
+  @override
+  Widget build(BuildContext context, ScopedReader watch) {
+    final theme = Theme.of(context);
+
+    return DevToolsDialog(
+      title: dialogTitleText(theme, 'State inspector configurations'),
```

Comment:
put 'State inspector configurations' into a static const on this class and use it here as well as for the tooltip above